config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+
+  get "user/:user_id/posts", to: "posts#index"
+  get "user/:user_is/posts/:id", to: "posts#show"
```

Comment:
- [ ] There's typo for the `posts#show` route, Pleaae fix it by replacing `/:user_is/` to `g/:user_id/`

Reply:
@thecodechaser !
Thank you for reviewing my code I will change it. :smiley: